flair-notifications/application-oss.yml

```diff
@@ -12,9 +12,11 @@ ssl:
   serverCert: certs/certChainFile.crt
   serverKey: certs/privateKeyFile.key
 mailService:
-  serive: gmail
+  serive: sendgrid
```

Comment:
is this parameter `serive` used at all anywhere?

Reply:
@johibkhan2 , what is this property used for?

Reply:
not used...will remove later point